### PR TITLE
Output sphinx warnings to stdout.

### PR DESCRIPTION
**Pull request by Cadair**
This allows us to silence stderr on CircleCI to make it more obvious what are actually warnings.

### CODE REVIEW

**Comment by Cadair** [comment]
Ok this didn't work.

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,16 +35,16 @@ jobs:
...
   html-docs:
-    environment:
-      - MAIN_CMD: "venv/bin/python setup.py"
-      - SETUP_CMD: "build_docs -w"
     docker:
       - image: circleci/python:3.6
     steps:
+      - restore_cache:
+          keys: sample-data-v1
+
```
Comment (dpshelio) [inline .circleci/config.yml]: Why this blank line?

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [submodule "astropy_helpers"]
 	path = astropy_helpers
-	url = https://github.com/astropy/astropy-helpers.git
+	url = https://github.com/Cadair/astropy-helpers.git
```
Comment (dpshelio) [inline .gitmodules]: Make an issue to remember (I know you will)

File: .circleci/config.yml (hunk 1 above)
Comment (Cadair) [reply .circleci/config.yml]: just to make it a little easier to separate the cache restore from the build part when reading the yaml.

**Pull request merged**